package.json

```diff
@@ -48,8 +48,8 @@
     "@commitlint/config-conventional": "7.6.0",
     "@commitlint/config-lerna-scopes": "7.6.0",
     "@commitlint/travis-cli": "7.6.1",
-    "@loopback/build": "1.5.3",
-    "@loopback/testlab": "1.2.8",
+    "@loopback/build": "1.5.4",
+    "@loopback/testlab": "1.2.9",
```

Comment:
I wonder why the dependencies are declared with fixed versions instead of `^` range.

Reply:
I think it's because we don't have the shopping example as downstream test in loopback-next, so if new release breaks the example we won't know.

Reply:
It's because of how we configured renovate-bot (see https://github.com/strongloop/loopback4-example-shopping/pull/93#issuecomment-489586795 and #104).

Reply:
@raymondfeng , @b-admike mentioned to me that `renovate-bot` goes into package.json and removes the `^` . So I just changed the versions to the latest.

Reply:
@b-admike did I paraphrase what you said properly?

Reply:
Yes @emonddr,  it'll remove them if we add the `^` back because of our settings.